modules/nf-core/fgbio/collectduplexseqmetrics/main.nf

```diff
@@ -0,0 +1,78 @@
+process FGBIO_COLLECTDUPLEXSEQMETRICS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-51891ad0b60843e4aade9cde2eb5d40c5ae92b80:72c944cdea5caff7f03b96034968ce2a4f1737bc-0':
```

Comment:
What is the mulled container needed for here?
There seems to be an inconsistency with the versions based on the tests, conda is reporting a different versions.yml md5sum.